journal.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
```

Comment:
Nice work guys! Could you please move the file to the `source/public` folder (this is where all of our code for the project will go, sorry if that wasn't clear)